webui/src/lib/components/auth/forms.jsx

```diff
@@ -9,15 +9,8 @@ import {SearchIcon} from "@primer/octicons-react";
 import {useAPI} from "../../hooks/api";
 import {Checkbox, DataTable, DebouncedFormControl, AlertError, Loading} from "../controls";
 
-const resolveEntityDisplayName = (ent) => {
-    // for users
-    if (ent?.email?.length) return ent.email;
-    // for groups
-    if (ent?.name?.length) return ent.name;
-    return ent.id;
-}
-
-export const AttachModal = ({ show, searchFn, onAttach, onHide, addText = "Add",
+
+export const AttachModal = ({ show, searchFn, onAttach, onHide, resolveEntityFN = (ent => ent.id), addText = "Add",
```

Comment:
Nit:
Should be `resolveEntityFN` (since we have `searchFn`).
I would also put but Fn params next to each other.